docs/examples/connect_series_to_class3D.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "21936b7f",
+   "metadata": {},
+   "source": [
+    "# Connect a volume series to Class3D\n",
+    "\n",
+    "This notebook connects each volume from a volume series (like those produced by 3D Var Display) to a 3D Class job. This saves the user the step of downloading a large volume series file, extracting it and uploading the volumes to the server, importing them, and manually connecting them to the job.\n",
+    "\n",
+    "```{note}\n",
+    "Since this script unzips the volume series itself, it must be run on a system with access to cryoSPARC's filesystem.\n",
```

Comment:
✨ Branding ✨ 

```suggestion
    "Since this script unzips the volume series itself, it must be run on a system with access to CryoSPARC's filesystem.\n",
```